frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -26,9 +27,31 @@ const ProvisionSourceDiskHelpMsg: React.FC<ProvisionSourceDiskHelpMsgProps> = ({
   const getStorageMsg = () => {
     switch (ProvisionSource.fromString(provisionSourceValue)) {
       case ProvisionSource.URL:
-        return <>Enter URL here or edit the mounted disk in the {storageBtn} step</>;
+        return (
+          <>
+            <Text> Enter URL here or edit the mounted disk in the {storageBtn} step</Text>
+            <Text>
+              For example, you can obtain the download link for {isUpstream ? 'Fedora' : 'RHEL'}{' '}
```

Comment:
These examples should be moved below the relevant field. So in this case it would be under the URL input.

Reply:
the URL field is right below it. changing the help providers for each field is not in the scope.
we should merge the changes and if we still find it wrong, we should open a new bug.